js/passwordManager/passwordCapture.js

```diff
@@ -48,11 +48,11 @@ const passwordCapture = {
   },
   handleRecieveCredentials: function (tab, args, frameId) {
     var domain = args[0][0]
```

Comment:
Shouldn't we be saving an origin or a URL into `currentDomain` (which later gets passed to `saveCredential`)? With this way, I think we're not recording the protocol.

In my branch I addressed this by passing the entire URL into the manager: https://github.com/minbrowser/min/compare/master...password-url-matching#diff-dfe869359e479c9fd76fbe8e10c5f46545b3ec71d7a029aafd096869489da6ed

Reply:
In the [passwordFill.js](https://github.com/minbrowser/min/pull/2562/files#diff-690430546b74ba1ee9ff083e7ac36602f2d263fd486073ba41545ea7256bf5dbR347) we pass the `origin` part of the url instead of the `hostname` which will give us the whole base url `https://sub.example.com` and that should be sufficient for the `saveCredential` funtionality.

Maybe the `domain` variable name is a little confusing, perhaps it needs to be something like `urlBase` or `urlOrigin`

Reply:
Got it, sorry for the delay. It sounds like something with "origin" would be a better name then.